src/3d/CMakeLists.txt

```diff
@@ -174,6 +171,17 @@ set(QGIS_3D_HDRS
   materials/qgssimplelinematerialsettings.h
   materials/qgstexturematerial.h
 
+  framegraph/qgsabstractrenderview.h
```

Comment:
New directories must be added to https://github.com/qgis/QGIS/blob/master/doc/CMakeLists.txt#L50

Reply:
done